src/current/_includes/v22.2/prod-deployment/healthy-crdb-memory.md

```diff
@@ -1 +1 @@
-**Expected values for a healthy cluster**: RSS minus Go Total and CGo Total should not exceed 100 MiB. Go Allocated should not exceed a few hundred MiB. CGo Allocated should not exceed the `--cache` size.
\ No newline at end of file
+**Expected values for a healthy cluster**: Go Allocated will depend on workload but should not exceed `--max-sql-memory` by more than 100%. CGo Allocated should not exceed the `--cache` size and CGo Total should not exceed the `--cache` size by more than 15%.
```

Comment:
First mention of `--cache` could be a link to the `cockroach start` docs where this flag's usage is defined in more detail

https://www.cockroachlabs.com/docs/v23.2/cockroach-start#general